src/components/PopupWithForm.js

```diff
@@ -6,6 +6,9 @@ export default class PopupWithForm extends Popup {
 
     this._popupForm = this._popupElement.querySelector(".modal__form");
     this._handleFormSubmit = handleFormSubmit;
+    this._popupElement = document.querySelector(popupSelector);
```

Comment:
`    this._popupElement = document.querySelector(popupSelector);
`

Please, delete it

**No** need to find the popup again in the inheritors of class `Popup` because the parent has already found it in its constructor and it’s `this._popupElement `. It’s available in any inheritor.